src/functions/module_array.ts

```diff
@@ -82,6 +82,62 @@ export const FLATTEN: AddFunctionDescription = {
   isExported: false,
 };
 
+// -----------------------------------------------------------------------------
+// FREQUENCY
+// -----------------------------------------------------------------------------
+export const FREQUENCY: AddFunctionDescription = {
```

Comment:
I wonder about this formula: Excel and google have different behavior.

On google, the results are always given in ascending order of classes.

while excel respects the order of the given classes

We could believe that it is practical at excel because you can give your classes according to a descending order.

But I find it weird, because in this case the last value of the result should also be placed at the beginning, which is not the case...

So, I wonder if we should not do like google for this formula --> always give the result in ascending order of classes

Reply:
In cases like this where the behaviour isn't the same on GSheet and Exel, I like to keep the excel bahavour sicne we support direct conversions o_spreadhseet <=> excel. 

Functionnaly I don't really care about what we do, but note that we can get Google behaviour by sorting our classes, but if we implement Google behaviour, we cannot have Excel behaviour.

Reply:
okay, so maybe add a comment in test to talk about compatibility with Google